diff --git a/src/omero/plugins/db.py b/src/omero/plugins/db.py
index aaa208b1c..c5c40ed7f 100644
--- a/omero/plugins/db.py
+++ b/omero/plugins/db.py
@@ -147,7 +147,7 @@ class DatabaseControl(BaseControl):
         value = p.communicate()[0]
         if not value or len(value) == 0:
             self.ctx.die(100, "Encoded password is empty")
-        return value.strip()
+        return value.strip().decode()
 
     def _copy(self, input_path, output, func, cfg=None):
             try:
