diff --git a/omeroweb/urls.py b/omeroweb/urls.py
index 71b2e05e8..48e5ae387 100755
--- a/omeroweb/urls.py
+++ b/omeroweb/urls.py
@@ -84,7 +84,12 @@ for app in settings.ADDITIONAL_APPS:
     except ImportError:
         pass
     else:
-        regex = '^(?i)%s/' % label
+        # see https://stackoverflow.com
+        #                 /questions/7580220/django-urls-how-to-map-root-to-app
+        if label == 'gallery':
+            regex = r'^'
+        else:
+            regex = '^(?i)%s/' % label
         urlpatterns.append(url(regex, include(urlmodule)))
 
 urlpatterns += [
